packages/alea-frontend/pages/api/send-forgot-email.ts

```diff
@@ -0,0 +1,44 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { executeAndEndSet500OnError } from './comment-utils';
+import nodemailer from 'nodemailer';
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const { email } = req.body;
+  const existingUser = (await executeAndEndSet500OnError(
+    `SELECT email FROM userInfo WHERE userId = ?`,
+    [email],
+    res
+  )) as any[];
+
+  //send email if user exist
+  if (existingUser.length === 1) {
```

Comment:
Should always send some response. if `existingUser` is not 1, return a different code.

If know response is sent, the promise of the caller is never resolved